Automattic/WP-Job-Manager
# Minor PHPCS fixes and GitHub Actions CI

### Changes proposed in this Pull Request

* Some pretty minor PHPCS issues snuck in. This fixes them.
* Ignoring date function issues for now. I'll be replacing them next in my PR for #919.
* Switches to GitHub Actions for CI.
* Updates dev script dependencies.

### Testing instructions

* Most non-whitespace changes are with the writepanels. Make sure the file upload field displays normally when editing a job in WP Admin.
